mentor_classifier/requirements.txt

```diff
@@ -10,5 +10,8 @@ PyYAML==5.4.1
 redis==3.5.3
 requests==2.25.1
 scikit-learn==0.24.2
-sentence-transformers==2.0.0
+sentence-transformers
```

Comment:
do we want sentence-transformers here or next PR?

Reply:
we still need sentence-transformers for the lr_transformer arch

Reply:
ok, in that case, just make sure we specify the version ,`sentence-transformers==2.0.0`...and we then need to include the changes to both Dockerfiles that do the cpu-only pip installs for py torch. Are you able to find those changes and add to this branch?